pkg/build/strategies/strategies.go

```diff
@@ -13,18 +13,13 @@ import (
 	utilstatus "github.com/openshift/source-to-image/pkg/util/status"
 )
 
-// GetStrategy decides what build strategy will be used for the STI build.
-// TODO: deprecated, use Strategy() instead
-func GetStrategy(client docker.Client, config *api.Config) (build.Builder, api.BuildInfo, error) {
-	return Strategy(client, config, build.Overrides{})
-}
-
 // Strategy creates the appropriate build strategy for the provided config, using
 // the overrides provided. Not all strategies support all overrides.
-func Strategy(client docker.Client, config *api.Config, overrides build.Overrides) (build.Builder, api.BuildInfo, error) {
+func Strategy(client docker.Client, config *api.Config) (build.Builder, api.BuildInfo, error) {
```

Comment:
Please revert - this will break openshift/builder

Reply:
How about define an empty build.Overrides{} in this function?

Reply:
This function is referenced by another project - changing signature breaks the API.